internal/config/conmonmgr/conmonmgr.go

```diff
@@ -0,0 +1,68 @@
+package conmonmgr
+
+import (
+	"path"
+	"strings"
+
+	"github.com/blang/semver"
+	"github.com/cri-o/cri-o/utils/cmdrunner"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+var versionSupportsSync = semver.MustParse("2.0.19")
```

Comment:
Is this value something we could (should?) move into a conf file?

Reply:
it's not really something that can be configured. it's a static property of how conmon was versioned. If an admin decided that conmon 1.0.0 supported this feature, crio would fail to create containers because conmon wouldn't recognize the flag